### ISSUE TITLE
Update monero-rs to work with hardfork v15

### ISSUE THREAD

**Issue by SWvheerden** [tari-project/tari#4489]
Recent Monero hardfork to v15 broke the merge mining.

**Comment by hansieodendaal** [comment]
Merge mining proxy error:
```
[tari_mm_proxy::proxy] ERROR Method "getblocktemplate" failed handling request: MergeMiningError { source: ValidationError("blocktemplate blob invalid") }
```

**Comment by SWvheerden** [comment]
Fixed in #4492
